config/locales/views/administrateurs/experts_procedures/fr.yml

```
@@ -0,0 +1,13 @@
+fr:
+  administrateurs:
+    experts_procedures:
+      index:
+        titles:
+          main: Experts invités sur %{libelle}
+          allow_invite_experts: "Autoriser les instructeurs à solliciter des experts invités"
+          allow_expert_messaging: "Autoriser les experts à acceder à la messagerie usager"
+          manage_procedure_experts: "Gérer les experts invités de la démarche"
+        descriptions:
+          allow_invite_experts : Si cette fonctionnalité est désactivée, les instructeurs ne pourront pas solliciter d’experts
+          allow_expert_messaging: Si cette fonctionnalité est désactivée, les experts ne pourront pas demander des informations aux usagers
+          manage_procedure_experts: Si cette fonctionnalité est activée, les instructeurs pourront uniquement inviter les experts de votre liste
```

J'ai eu du mal à comprendre le sens de ces descriptions, je propose de profiter de cette PR pour les simplifier et les mettant à l'affirmative et au présent :

```suggestion
          allow_invite_experts : Lorsque cette fonctionnalité est active, les instructeurs peuvent solliciter les experts
          allow_expert_messaging: Lorsque cette fonctionnalité est active, les experts peuvent demander des informations aux usagers
          manage_procedure_experts: Lorsque cette fonctionnalité est active, les instructeurs ne peuvent uniquement solliciter les experts de votre liste
```